examples/run/cuda/seq_example_cuda.cpp

```diff
@@ -176,6 +176,10 @@ int seq_run(const std::string& detector_file, const std::string& cells_dir,
             }
             float matching_rate = float(n_match) / seeds.get_headers()[0];
             std::cout << "event " << std::to_string(event) << std::endl;
+            std::cout << " number of seeds (cpu): " << seeds.get_headers()[0]
+                      << std::endl;
+            std::cout << " number of seeds (cuda): "
+                      << seeds_cuda.get_headers()[0] << std::endl;
```

Comment:
Just for clarification, and to show my ignorance: In the CPU implementation the seed container is binned according to the geometrical grid, right? But the CUDA code handles all seeds in the event inside of a single bin. And that's why we only need to consider the 0th element of the headers, right? :thinking:

I was pondering for a moment why we would even need to use jagged vectors for the seeds if we only have "a single bin" in the container. But then I remembered that the CPU code probably makes use of the jagged vector nature of the container...

Reply:
> But the CUDA code handles all seeds in the event inside of a single bin. And that's why we only need to consider the 0th element of the headers, right? 

Right. And I agree that it should be vector rather than jagged_vector. 
The simplest way to make it vector is allocating the exact size to the vector and populate the seeds in it.
and that's what I tried to do in #110 which allocates the exact size to multiplet container. 
I managed to do that for doublet and triplet container but decided not to do that for seed container for some technical reasons. 

However, that's still on my to-do list; I hope I will fix it in the near future